clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -718,9 +718,14 @@ DPCT_ENUM_OPTION(
             "logical-group", int(ExperimentalFeatures::Exp_LogicalGroup),
             "Experimental helper function used to logically group work-items.",
             false),
+        DPCT_OPTION_ENUM_VALUE("root-group",
+                               int(ExperimentalFeatures::Exp_RootGroup),
+                               "Experimental extension that allows use of root "
+                               "group class and relative API.\n",
+                               false),
         DPCT_OPTION_ENUM_VALUE(
             "nd_range_barrier", int(ExperimentalFeatures::Exp_NdRangeBarrier),
-            "Experimental helper function used to help cross-group "
+            "DEPRECATED : Experimental helper function used to help cross-group "
```

Comment:
how about adding more msg,  like using root-group instead?

Reply:
"DEPRECATED : Experimental helper function used to help "
            "cross-group "
            "synchronization during migration. Please try to remigrate with "
            "option: --use-experimental-features=root-group\n"